src/pipecat/processors/frame_processor.py

```diff
@@ -190,6 +214,16 @@ async def wait_for_task(self, task: asyncio.Task, timeout: Optional[float] = Non
             raise Exception(f"{self} TaskManager is still not initialized.")
         await self._task_manager.wait_for_task(task, timeout)
 
+    def start_watchdog(self):
+        if not self._task_manager:
+            raise Exception(f"{self} TaskManager is still not initialized.")
+        self._task_manager.start_watchdog(asyncio.current_task())
```

Comment:
Since this validation is already handled inside `get_task_manager()`, could we simplify those by just calling:
```suggestion
        self.get_task_manager().start_watchdog(asyncio.current_task())
```

Reply:
The same for the other methods.

Reply:
Good idea! Done!